src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -1052,10 +1052,12 @@ export class IssueReporter extends Disposable {
 			}, extension.name);
 		};
 
-		const extensionsSelector = this.getElementById('extension-selector');
+		const extensionsSelector = this.getElementById('extension-selector')! as HTMLSelectElement;
 		if (extensionsSelector) {
 			const { selectedExtension } = this.issueReporterModel.getData();
 			reset(extensionsSelector, $<HTMLOptionElement>('option'), ...extensionOptions.map(extension => makeOption(extension, selectedExtension)));
+			extensionsSelector.replaceChild(this.makeOption('', localize('selectExtension', "Select extension"), true), extensionsSelector.firstChild!);
```

Comment:
these two lines can be like:
```
reset(
  extensionsSelector,
  $<HTMLOptionElement>('option'),
  this.makeOption('', localize('selectExtension', "Select extension"), true)
  ...extensionOptions.map(extension => makeOption(extension, selectedExtension))
);
```
right? something like that

Reply:
That's what I tried at first, but the reason I went with the replace was because `reset........extensionOptions.map(extension => makeOption(extension, selectedExtension)` was writing an extra empty option (like below).

![Screenshot 2023-09-19 at 10 55 52 AM](https://github.com/microsoft/vscode/assets/54879025/06b03553-8499-47ba-896a-66e497ad46f3)

Reply:
But how? Where does the empty option come from?

Reply:
is it `  $<HTMLOptionElement>('option'),` lol

Reply:
you are so right LOL I didn't even see the comma there for some reason I just thought it was part of the call